src/integrationtests/resources/com/aws/greengrass/integrationtests/privateKey.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded private key. This practice can expose your application to serious security risks, including unauthorized decryption of sensitive data and potential system intrusion. Possible remediation approaches are:
	 1. Manage secret data using environment variables:
		 a) Identify the secret data and store that data in environment variables on your local and production server.
		 b) Replace the secrets with references to environment variables.
	 2. Use AWS Secrets Manager to store and access secrets:
		 a) Visit Secrets Manager on the AWS Management Console.
		 b) Choose the secret type on the console page and follow the instructions.
		 c) Use the suggested code samples to retrieve the secret.

To learn more about safe key management practices, see [OWASP key management cheat sheet](https://www.owasp.org/index.php/Cryptographic_Storage_Cheat_Sheet).